aws-proofs/vm-steps.sh

```diff
@@ -91,8 +91,14 @@ do_run_tests || FAIL=1
 echo
 echo "Stats:"
 ~/ci-actions/aws-proofs/kernel-sloc.sh
-echo ""
-cd l4v; ~/ci-actions/aws-proofs/sorry-count.sh; cd ..
+
+# sorry-count script lives in l4v repo; guard against branches where it might be gone
+SORRY_COUNT=misc/stats/sorry-count.sh
+if [ -x "l4v/${SORRY_COUNT}" ]
+then
+  echo ""
+  cd l4v; "./${SORRY_COUNT}"; cd ..
```

Comment:
```suggestion
  cd l4v; "${SORRY_COUNT}" ; cd ..
```
No sure what shells or environment this has to work with, but is the `./` really needed?

Reply:
It is definitely better shell style to always anchor the command, otherwise you don't know what you are calling. With the current value of `SORRY_COUNT` it is redundant, but if that value ever changes to a bare command, the shell could be selecting something from the PATH.